specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/virtualWan.json

```diff
@@ -646,7 +646,315 @@
+          "final-state-via": "location"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/vpnServerConfigurations": {
+      "get": {
+        "operationId": "VpnServerConfigurations_ListByResourceGroup",
+        "x-ms-examples": {
+          "VpnServerConfigurationListByResourceGroup": {
+            "$ref": "./examples/VpnServerConfigurationListByResourceGroup.json"
+          }
+        },
+        "description": "Lists all the vpnServerConfigurations in a resource group.",
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The resource group name of the VpnServerConfiguration."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. Returns the details of all the vpnServerConfigurations in the resource group.",
+            "schema": {
+              "$ref": "#/definitions/ListVpnServerConfigurationsResult"
+            }
+          },
+          "default": {
+            "description": "Error.",
+            "schema": {
+              "$ref": "./network.json#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Network/vpnServerConfigurations": {
+      "get": {
+        "operationId": "VpnServerConfigurations_List",
+        "x-ms-examples": {
+          "VpnServerConfigurationList": {
+            "$ref": "./examples/VpnServerConfigurationList.json"
+          }
+        },
+        "description": "Lists all the VpnServerConfigurations in a subscription.",
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. Returns the details of all the VpnServerConfigurations in the subscription.",
+            "schema": {
+              "$ref": "#/definitions/ListVpnServerConfigurationsResult"
+            }
+          },
+          "default": {
+            "description": "Error.",
+            "schema": {
+              "$ref": "./network.json#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },	
+	
+	"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualWans/{virtualWANName}/vpnConfiguration": {
```

Comment:
POST actions are expected to be named with a leading verb, such as:
`exportVPNConfiguration` or `downloadVPNConfiguration`

Reply:
Sorry for adding confusion. This api is not needed. Removed it.